src/components.d.ts

```diff
@@ -23,7 +26,7 @@ export namespace Components {
 
   interface Card3Form {
     'state': "failed" | "succeeded" | "processing" | "created";
-    'submit': (method: string, amount: number, currency: Currency, order?: string | undefined, reference?: string | undefined, description?: string | undefined) => void;
+    'submit': (method: import("/home/smika/versioned/github/certitrade/card3-form/node_modules/@certitrade/card3-model/dist/Method").Method, amount: number, currency: Currency, order?: string | undefined, reference?: string | undefined, description?: string | undefined) => Promise<Payment>;
```

Comment:
Hardcoded repo?

Reply:
Oops, good catch.

Reply:
It's in the autogenerated `Components.d.ts` file. Maybe time to try to remove it from the repo?

Reply:
Yes. Maybe.